src/encoder.rs

```diff
@@ -97,6 +98,7 @@ impl<T: Pixel> ReferenceFramesSet<T> {
   }
 }
 
+#[wasm_bindgen]
```

Comment:
Adding the wasm dependency is not acceptable. either rebind or add wasm_bindgen to the [noop_proc_macro](https://crates.io/crates/noop_proc_macro) crate.

Reply:
For both `rav1e` and `v_frame`, I guess?